recipes/libprotobuf-mutator/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class LibProtobufMutatorConan(ConanFile):
+    name = "libprotobuf-mutator"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/libprotobuf-mutator"
+    description = "A library to randomly mutate protobuffers."
+    topics = ("test", "fuzzing", "protobuf")
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def requirements(self):
+        self.requires("protobuf/3.17.1")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+        tools.replace_in_file(
```

Comment:
```suggestion

    def _patch_sources(self):
        tools.replace_in_file(
```

You can't patch sources under source method. Try to keep all files there untouched, because in case you need to manipulate anything there, Conan will download all files again and again.